src/app/core/header/header.component.ts

```diff
@@ -1,16 +1,21 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'lms-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.css']
 })
 export class HeaderComponent {
-  constructor() { }
+  constructor(private router: Router ) {}
   loggedInUserName: string = '';
 
   ngOnInit() {
     let loggedInUserobj = JSON.parse(sessionStorage.getItem('loggedInUser')!);
     this.loggedInUserName = loggedInUserobj.name;
   }
+  logout() {
+    sessionStorage.removeItem('loggedInUser');
```

Comment:
we need to remove/clear all session storage variables while logout, not only loggedInUser, so use sessionStorage.clear();